app/serializers/component_serializer/figure_component_serializer.rb

```diff
@@ -8,7 +8,7 @@ class FigureComponentSerializer < BaseComponentSerializer
     # @param [boolean] tab_index an optional parameter which adds a tab_index property to data.
     # @param [Array<Hash>] source_info is a hash containing the any source tag information, up to three key/value pairs. Including: media, srcset and additional srcset.
     # @param [Array<Hash>] img is a hash containing three key/value pairs to go in the img tag. One string of text or translation key for the alt label, one with any data from the backend to go in the alt label and the link for the src.
-    # @param [Array<Hash>] figcap is a hash containing two key/value pairs. One for the content strign or translation key of the figure caption and one for any data it might need from the backend.
+    # @param [String/Array<Hash>] figcap  can be a string or translation key, or it can be a hash containing a translation key and data to be interpolated. To create the hash you must use the ContentDataHelper
```

Comment:
Should is be `[String|Array<Hash>]`? Might be worth check the yard cheatsheet

Reply:
Oh that is updated in the next pull request